src/language.js

```diff
@@ -88,6 +100,21 @@ const error = {
     `Something went wrong. The Project ${projectName}, may not be ready to be used offline. Please try again.`,
   getProjectTurnOffOfflineReadyFailure: (projectName) =>
     `Something went wrong. The Project ${projectName}, has not been removed from being offline-ready.`,
+  getPullSyncErrorMessage: (projectName) => (
```

Comment:
im sneaking some minor formatting into the language file. I think this is passable since its just some bolding and avoids chopping up sentences into parts that wont necessarily work in other languages. I wouldnt take it much further than that though...